packages/api-derive/test/e2e/rx.spec.ts

```diff
@@ -23,49 +28,190 @@ describe.skip('derive e2e', () => {
+      it('It returns an object with all relevant balance information of an account', async (done) => {
+        api.derive.balances.all(ID).subscribe((balances) => {
+          expect(balances).toEqual(expect.objectContaining({
+            accountId: expect.any(AccountId),
+            availableBalance: expect.any(Balance),
+            freeBalance: expect.any(Balance),
+            lockedBalance: expect.any(Balance),
+            reservedBalance: expect.any(Balance),
+            vestedBalance: expect.any(Balance),
+            votingBalance: expect.any(Balance)
+          }));
+          done();
+        });
+      });
+    });
+
+    describe('fees', () => {
+      it('fees: It returns an object with all relevant fees of type BN', async (done) => {
+        api.derive.balances.fees().subscribe((fees) => {
+          expect(fees).toEqual(expect.objectContaining({
+            creationFee: expect.any(BN),
+            existentialDeposit: expect.any(BN),
+            transactionBaseFee: expect.any(BN),
+            transactionByteFee: expect.any(BN),
+            transferFee: expect.any(BN)
+          }));
+          done();
+        });
+      });
     });
   });
 
-  it('returns the intentions with balances', async (done) => {
-    api.derive.staking.intentionsBalances().subscribe((balances) => {
-      expect(Object.keys(balances as object)).not.toHaveLength(0);
-      done();
+  describe('derive.chain', () => {
+    describe('bestNumber', () => {
+      it('Get the latest block number', async (done) => {
+        api.derive.chain.bestNumber().subscribe((blockNumber) => {
+          expect(blockNumber instanceof BlockNumber).toBe(true);
+          expect((blockNumber as BlockNumber).gten(0)).toBe(true);
+          done();
+        });
+      });
+    });
+
+    describe('bestNumberFinalized', () => {
+      it('Get the latest finalised block number', async (done) => {
+        api.derive.chain.bestNumberFinalized().subscribe((blockNumber) => {
+          expect(blockNumber instanceof BlockNumber).toBe(true);
+          expect((blockNumber as BlockNumber).gten(0)).toBe(true);
+          done();
+        });
+      });
+    });
+
+    describe('bestNumberLag', () => {
+      it('lag between finalised head and best head', async (done) => {
+        api.derive.chain.bestNumberLag().subscribe((numberLag) => {
+          expect(numberLag instanceof BlockNumber).toBe(true);
+          expect((numberLag as BlockNumber).gten(0)).toBe(true);
+          done();
+        });
+      });
+    });
+
+    describe('getHeader', () => {
+      it('gets a specific block header and extended with it\`s author', async (done) => {
+        api.derive.chain.getHeader().subscribe((headerExtended) => {
+          expect(headerExtended).toEqual(expect.arrayContaining([]));
+          done();
+        });
+      });
+    });
+
+    describe('subscribeNewHead', () => {
+      it('gets an observalbe of the current block header and it\'s author', async (done) => {
```

Comment:
(nit) observable (This one is like commenting on my own PRs, I always seem to have some spelling mishap somewhere...)

Reply:
I had to read it 3 time until I saw the `salbe`... :|